### ISSUE TITLE
Availability/support for ARM64 VPS

### ISSUE THREAD

**Issue by thekyriarchy** [scaleway/image-fedora#34]
I'd like to use Fedora on the 64-bit ARMv8 VPS

**Comment by halmartin** [comment]
Hi @thekyriarchy !

We are working on this. I hope to have some good news for you in the coming weeks.

**Comment by halmartin** [comment]
I have the Fedora 28 Beta booting on aarch64 in our lab. :tada:

```
[root@compassionate-mcnulty ~]# uname -a
Linux compassionate-mcnulty 4.16.4-300.fc28.aarch64 #1 SMP Tue Apr 24 21:59:52 UTC 2018 aarch64 aarch64 aarch64 GNU/Linux
[root@compassionate-mcnulty ~]# cat /etc/fedora-release 
Fedora release 28 (Twenty Eight)
```
---

I can say that we will not be able to provide images for Fedora 28 immediately after the Fedora 28 GA, as there are still some major issues we need to resolve.

I will provide updates on support for Fedora 28 on aarch64 as we progress.

**Comment by heyakyra** [comment]
Awesome! Thanks for that update. Are the remaining issues being tracked somewhere I can follow along?

**Comment by halmartin** [comment]
@heyakyra No, sorry, the issue tracking for pre-release images isn't public. We are still working on this and I will update you when I have more news.

**Comment by halmartin** [comment]
@heyakyra I just wanted to provide an update for you on Fedora 28 for aarch64.

We are currently blocked on an upstream bug in Fedora 28: https://bugzilla.redhat.com/show_bug.cgi?id=1576573

Currently I don't have an ETA for when Red Hat will push a fix. I'll be back with more details when we're able to proceed.

**Comment by halmartin** [comment]
@heyakyra hi, sorry for the delay in this. We haven't forgotten about Fedora 28 on aarch64! We are working on some outstanding issues, which aren't major but we feel should be resolved before releasing the image to clients.

In case this doesn't get resolved before F29 is released, we'll be sure to keep up with the releases :)

**Comment by alcir** [comment]
hello, any news?
